hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/client/TestQuorumJournalManager.java

```diff
@@ -1171,6 +1179,47 @@ public void testSelectViaRpcAfterJNRestart() throws Exception {
     }
   }
 
+  /**
+   * Tests to throw an exception if the jn maintenance nodes exceeds half of the journalnode number.
```

Comment:
nit: 

"if the jn maintenance nodes exceeds half of the journalnode number." -> "when more than half of the journal nodes are in maintenance mode".